test/tools/Modules/HelpersDebugger/HelpersDebugger.psd1

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+@{
+    RootModule = 'HelpersDebugger.psm1'
+
+    ModuleVersion = '1.0'
+
+    GUID = '37a454d7-8acd-40e6-8a2c-43c9d46b1b0c'
+
+    CompanyName = 'Microsoft Corporation'
+
+    Copyright = 'Copyright (c) Microsoft Corporation. All rights reserved.'
+
+    Description = 'Helper module for Pester tests that automate the debugger'
+
+    FunctionsToExport = @(
+        'Register-DebuggerHandler'
+        'ShouldHaveExtent'
+        'ShouldHaveSameExtentAs'
+        'Test-Debugger'
+        'Unregister-DebuggerHandler'
+    )
+
+    CmdletsToExport = @()
+
+    AliasesToExport = @()
```

Comment:
We can remove CmdletsToExport  and AliasesToExport.

Reply:
Do they default to an empty array if they are not present? Inclusion of these values as empty arrays used to be a best practice so that command discovery performed more efficiently. Further, this module works in Windows PowerShell as well as PowerShell Core, so I don't think the removal of these values is a good idea. Leaving them in allows the module to be used to run tests in Windows PowerShell as well for comparison while following the best practices required for module use there.